homeassistant/components/bmw_connected_drive/button.py

```diff
@@ -69,13 +72,6 @@ class BMWButtonEntityDescription(ButtonEntityDescription):
         icon="mdi:crosshairs-question",
         remote_function=lambda vehicle: vehicle.remote_services.trigger_remote_vehicle_finder(),
     ),
-    BMWButtonEntityDescription(
-        key="refresh",
-        translation_key="refresh",
```

Comment:
Don't forget to remove the key from strings.json

Reply:
Also needs to be removed from `services.yaml`

Reply:
Removed from `strings.json`. But there is no `services.yaml` anymore, or am I wrong?

Reply:
@gjohansson-ST BMW doesnt have services?

Reply:
We had, but moved all to buttons/switches/numbers :)

Reply:
My mistake. Confused it with a different one. Thanks